strapi/strapi
# Feat(releases): Ability to schedule a release

### What does it do?

- Adds the ability to schedule a release
- Timezone selected will default to the users location

### How to test it?
When creating or editing a release, you should see a modal to edit the name and the “Schedule release” checkbox should be available:
- When schedule release is checked, you see 3 fields below: date, time and timezone. User should not be able to save unless all fields are selected.
- When schedule release is unchecked, you don’t see scheduling fields and user should be able to save.
